securesignals-gma-devapp/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,8 @@
+<resources>
```

Comment:
Nit: Style shouldn't be required.

Reply:
Android disagrees when I removed it:

<img width="1965" alt="Screenshot 2023-04-19 at 11 36 00 AM" src="https://user-images.githubusercontent.com/989928/233142037-311d1149-209a-4f96-b1d1-db6c1f4ccfc7.png">